src/codeu/chat/client/Login.java

```diff
@@ -0,0 +1,44 @@
+package codeu.chat.client;
+
+import java.awt.*;
+import java.awt.event.*;
+import javax.swing.*;
+import javax.swing.border.*;
+
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import codeu.chat.common.BasicView;
+import codeu.chat.common.Conversation;
+import codeu.chat.common.ConversationSummary;
+import codeu.chat.common.LogicalView;
+import codeu.chat.common.Message;
+import codeu.chat.common.NetworkCode;
+import codeu.chat.common.User;
+import codeu.chat.util.Logger;
+import codeu.chat.util.Serializers;
+import codeu.chat.util.Time;
+import codeu.chat.util.Uuid;
+import codeu.chat.util.connections.Connection;
+import codeu.chat.util.connections.ConnectionSource;
+import codeu.chat.client.ClientUser;
+
+public class Login {
+    private static User user;
```

Comment:
Why are these static? I think that's very much not what you wanted to do. If two login requests happen concurrently, the second request overrides the ClientUser object for the first.